app/src/components/navigation-bar.tsx

```diff
@@ -47,7 +51,7 @@ export function NavigationBar({
 
     // change language in URL without reloading page
     const newPath = location.pathname.replace(/^\/[A-Za-z]+/, `/${language}`);
-    history.replaceState("", "", newPath);
+    history.replaceState(null, "", newPath);
```

Comment:
I think we should also consider making changes to all our parts using `history.replaceState` in the future too